content/applications/general/email_communication/email_servers.rst

```diff
@@ -252,3 +166,68 @@ Now, go to :menuselection:`Email Marketing --> Settings` and enable :guilabel:`D
+
+.. _email_communication/inbound_messages:
+
+Manage inbound messages
+=======================
+
+Odoo relies on generic email aliases to fetch incoming messages.
+
+* **Reply messages** of messages sent from Odoo are routed to their original discussion thread (and
+  to the inbox of all its followers) by the catchall alias (**catchall@**).
+* **Bounced messages** are routed to **bounce@** in order to track them in Odoo. This is especially
+  used in `Odoo Email Marketing <https://www.odoo.com/page/email-marketing>`__ to opt-out invalid
+  recipients.
+* **Original messages**: Several business objects have their own alias to create new records in
+  Odoo from incoming emails:
+
+  * Sales Channel (to create Leads or Opportunities in `Odoo CRM <https://www.odoo.com/page/
+    crm>`__),
+  * Support Channel (to create Tickets in `Odoo Helpdesk <https://www.odoo.com/page/helpdesk>`__),
+  * Projects (to create new Tasks in `Odoo Project <https://www.odoo.com/page
+    /project-management>`__),
+  * Job Positions (to create Applicants in `Odoo Recruitment <https://www.odoo.com/page
+    /recruitment>`__),
+  * etc.
+
+Depending on the mail server, there might be several methods to fetch emails. The easiest and most
+recommended method is to manage one email address per Odoo alias in the mail server.
+
+* Create the corresponding email addresses in the mail server (catchall@, bounce@, sales@, etc.).
+* Set the :guilabel:`Alias Domain` name in :menuselection:`Settings --> General Settings -->
+  Discuss`.
+* If the database's hosting type is Odoo on-premise, create an :guilabel:`Incoming Mail Server` in
+  Odoo for each alias. This can be done from the General Settings as well. Fill out the form
+  according to the email provider's settings. Leave the :guilabel:`Actions to Perform on Incoming
+  Mails` field blank. Once all the information has been filled out, click on :guilabel:`TEST &
+  CONFIRM`.
+
+  .. image:: email_servers/incoming-server.png
+     :align: center
+     :alt: Incoming mail server configuration on Odoo.
+
+* If the database's hosting type is Odoo Online or Odoo.sh, redirecting or forwarding incoming
+  messages to Odoo's domain name instead of the external email server is recommended. That way,
+  incoming messages can be received without delay. Odoo Online only fetches incoming messages of
+  external servers once every hour. Redirections for all email addresses should be set to Odoo's
+  domain name in the email server (e.g. *catchall\@mydomain.ext* to *catchall\@mycompany.odoo.com*).
+
+All the aliases are customizable in Odoo. Object aliases can be edited from their respective
+configuration view.
+
+.. tip::
+   To edit catchall and bounce aliases, first activate the :ref:`developer mode <developer-mode>`.
+   Then, go to :menuselection:`Settings --> Technical --> Parameters --> System Parameters` to
+   customize the aliases (*mail.catchall.alias* & *mail.bounce.alias*).
```

Comment:
tip but also be careful,
you might lose some emails if you do this, 
the change needs to be done prior to go live, once you go live, if customer reply to a catchall that been changed, well, the system will not recognise the local-part anymore.

Reply:
Good point, I've added this in.